src/services/userDataProcessingService.ts

```diff
@@ -95,4 +98,40 @@ export default class UserDataProcessingService {
       );
     });
   };
+
+  /**
+   * Abort the user data processing of a specific user.
+   */
+  public readonly abortUserDataProcessing = async (
+    user: User,
+    userDataProcessingChoiceParam: UserDataProcessingChoice
+  ): Promise<
+    // tslint:disable-next-line: max-union-size
+    | IResponseErrorInternal
+    | IResponseErrorTooManyRequests
+    | IResponseErrorNotFound
+    | IResponseErrorValidation
+    | IResponseErrorConflict
+    | IResponseSuccessAccepted
+  > => {
+    const client = this.apiClient.getClient();
+    return withCatchAsInternalError(async () => {
+      const validated = await client.abortUserDataProcessing({
+        choice: userDataProcessingChoiceParam,
+        fiscal_code: user.fiscal_code
+      });
+
+      return withValidatedOrInternalError(validated, response =>
+        response.status === 202
+          ? ResponseSuccessAccepted()
+          : response.status === 404
+          ? ResponseErrorNotFound("Not Found", "User data processing not found")
+          : response.status === 409
+          ? ResponseErrorConflict("Cannot abort request")
```

Comment:
```suggestion
          ? ResponseErrorConflict("Cannot abort user data processing request")
```

request may refer to the HTTP request